rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec

```diff
@@ -526,6 +535,14 @@ if [ -f %{_localstatedir}/tmp/wazuh.restart ]; then
   fi
 fi
 
+if [ -d %{_localstatedir}/logs/ossec ]; then
+  rm -rf %{_localstatedir}/logs/ossec/
+fi
+
+if [ -d %{_localstatedir}/queue/ossec ]; then
+  rm -rf %{_localstatedir}/queue/ossec/
+fi
+
```

Comment:
Update also the files section like 4.2.0 SPEC

Reply:
That issue was fixed in this PR https://github.com/wazuh/wazuh-packages/pull/643